site2/docs/concepts-architecture-overview.md

```diff
@@ -150,3 +150,5 @@ from pulsar import Client
 
 client = Client('pulsar://pulsar-cluster.acme.com:6650')
 ```
+
+Note that in Pulsar, each topic is handled by only one broker. Therefore, for creating, reading, updating and deleting topics, requests are sent to any broker initially. If the broker finds that it cannot handle the request for this topic, it redirects the client to the correct broker. 
```

Comment:
> Note
> In Pulsar, each topic is handled by only one broker. Requests for creating, reading, updating and deleting topics are sent to the broker owner initially. If the broker owner cannot handle the request for this topic, it redirects the request to  another/ the correct broker.

Reply:
Thanks - I took your feedback and made a few tweaks to it.